### DISCUSSION TITLE
PDPERFPLUGIN, what???

### DISCUSSION THREAD

**Discussion by makemeone-prog** [praydog/REFramework#1105] [General Questions]
basically the title, where in the world are the releases that are compatible with the upscaling mods? absolutely none of the nightly builds work and i know there was one when i see tons of people talking about it and using it https://github.com/praydog/REFramework/commit/f18adbe05ae937bdb01f788e6531fefd012ffed1

there is talk of such build however the links to download them say "expired"

**Comment by makemeone-prog** [comment]
one very famous youtube/gamerants article goes on explaining how to add dlss to resident evil 4 but links to a build that doesn't offer support for the game, however the latest builds for re4 seemingly work for me. 

it's just that stable builds and the nightly builds are missing support for resident evil 2 3 and 7.

**Comment by praydog** [comment]
Builds that have the label `pd-upscaler` next to them: https://github.com/praydog/REFramework/actions

**Reply by makemeone-prog** [reply]
you're a gdsend!
i'd love to buy you a cofee
